easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-4.2.1-GCCcore-11.3.0.eb

```diff
@@ -27,7 +27,8 @@ dependencies = [
     ('freetype', '2.12.1'),
 ]
 
-preconfigopts = "./autogen.sh && "
+# define $GTKDOCIZE to 'echo' as workaround for https://github.com/easybuilders/easybuild-easyconfigs/issues/16567
+preconfigopts = "GTKDOCIZE=echo ./autogen.sh && "
```

Comment:
But isn't this a more general problem for Autotools? I wonder if the fix should not be in the `ConfigureMake` easyblock since people may filter out Autotools.

Reply:
The problem is related to the fact that `autogen.sh` has a hardcoded check for `gtkdocize`, so I think this is mostly specific for HarfBuzz, otherwise we would have seen this already?

Reply:
Ah, sorry `autogen.sh` is local, not from Autotools, my mistake